class Failure implements Exception {}
// Criei um error "Pr√≥prio" assim deixando o Exception de lado.